requirements.txt

```diff
@@ -10,8 +10,8 @@ Cheroot==8.6.0
 pycountry==22.3.5
 # plugins
 # EdgarRenderer plugin
-NumPy==1.23.3
-Matplotlib==3.6.0
+NumPy==1.21.6
```

Comment:
The version range in pyproject.toml should also be updated

Reply:
Discussed offline. pip should handle installing a version from the range in pyproject.toml that's compatible with the current python runtime